translations/mod-fqm-manager/en.json

```
@@ -308,5 +308,20 @@
   "entityType.drv_holdings_record_details.holdings_temporary_location": "Holdings temporary location",
   "entityType.drv_holdings_record_details.holdings_temporary_location_id": "Holdings temporary location ID",
   "entityType.drv_holdings_record_details.holdings_statistical_code_ids": "Holdings statistical code ID list",
-  "entityType.drv_holdings_record_details.holdings_statistical_codes": "Holdings statistical code list"
+  "entityType.drv_holdings_record_details.holdings_statistical_codes": "Holdings statistical code list",
+
+  "entityType.src_organization_type": "Organization types",
+  "entityType.src_organization_type.id": "ID",
+  "entityType.src_organization_type.organization_types_name": "Type",
+  "entityType.drv_organization_details" : "Organizations - vendor info",
+  "entityType.drv_organization_details.last_updated": "Org last updated",
+  "entityType.drv_organization_details.code": "Org Code",
+  "entityType.drv_organization_details.name": "Org Name",
+  "entityType.drv_organization_details.organization_status": "Org status",
+  "entityType.drv_organization_details.description": "Org Description",
+  "entityType.drv_organization_details.accounting_code": "Org accounting code",
+  "entityType.drv_organization_details.organization_type_ids": "Org type ID list",
+  "entityType.drv_organization_details.organization_type_name": "Org type list",
```

It may be good to clarify this is a list of names, not IDs (to disambiguate from the column above)
```suggestion
  "entityType.drv_organization_details.organization_type_name": "Organization type names",
```
Not sure if `names`, `labels`, etc, would be most appropriate